### PR TITLE
recreated content for myconnect

### PR DESCRIPTION
**Pull request by ramavavi**
The issue address is : https://github.com/episphere/connect/issues/727

The changes made in the code are:

1. Removed the api call for the functions and added the html and css to it. 
2.  Added responsiveness to the pages to look good in all the devices. 
3. Remove the extra spaces and adding alignment to the content.
4. Reducing the size of the images to fit properly.

### CODE REVIEW

File: js/pages/homePage.js [hunk 1]
```diff
@@ -167,32 +167,485 @@ export const whereAmIInDashboard = async () => {
...
+
         </div>
         <div class="col-1">
         </div>
         </div>
         `
-    renderSyndicate("https://api.digitalmedia.hhs.gov/api/v2/resources/media/19350/syndicate.json?stripStyles=false&stripScripts=false&stripBreaks=false&stripImages=false&stripClasses=false&stripIds=false&displayMethod=undefined&autoplay=false","connectBody", 'expectations')
+    // renderSyndicate("https://api.digitalmedia.hhs.gov/api/v2/resources/media/19350/syndicate.json?stripStyles=false&stripScripts=false&stripBreaks=false&stripImages=false&stripClasses=false&stripIds=false&displayMethod=undefined&autoplay=false","connectBody", 'expectations')
     window.scrollTo(0, 0);
     let sections = document.getElementsByTagName('h2')
```
Comment (JoeArmani) [inline js/pages/homePage.js:651]: please remove the console.log() ~line 652

**Review by JoeArmani — CHANGES_REQUESTED** [review]
The responsiveness looks good.
Checked:
√-About page
√-What to Expect page
√-Privacy page

General Issue 1:
I'm seeing quite a few CORS errors related to the Montserrat font from the new stylesheets that were imported. Have you noticed these? Will they be an issue?
Screenshot 2023-08-09 at 5 18 48 PM
Screenshot 2023-08-09 at 5 18 09 PM

General Issue 2:
I noticed a Layout Shift (double loading) effect when loading About, What to Expect, and Privacy pages. First, you see the heading and two bullet points with no text. Then the page loads. What can be done about this? I think it could be detrimental to user experience. I double checked and this doesn't occur on the dev branch. When I commented out the newly added stylesheets, I see this (example of the double bullet points, but it doesn't get hidden):
Screenshot 2023-08-09 at 5 29 35 PM

General Issue 3:
Requesting changes on the code styling. Please format the entirety of:
`renderHomeAboutPage()`
`renderHomeExpectationsPage()`
`renderHomePrivacyPage()`
So that the code maintainable & extensible in the future. Here are a few screenshots of how the code looks on my end just in case it's an issue with how my import worked when I checked out your branch. If there are discrepancies on your end, please let me know.
Screenshot 2023-08-09 at 5 00 31 PM
Screenshot 2023-08-09 at 5 00 48 PM
Screenshot 2023-08-09 at 5 01 11 PM

File: js/pages/homePage.js [hunk 2]
```diff
@@ -167,107 +167,479 @@ export const whereAmIInDashboard = async () => {
...
+                    <li class="page-options--print">
+                      <a title="Print" aria-label="Print" href="#"></a>
+                    </li>
+                    <li class="page-options--email">
+                      <a title="Email" aria-label="Email" href="#"></a>
+                    </li>
+                </ul>
+              </div>
+              <!-- END PAGE OPTIONS -->
+          </div>
```
Comment (JoeArmani) [inline js/pages/homePage.js:329]: remove this div. These are not in use and they're rendering as bullet points with no text. In the current format, this disappears when the stylesheet is fetched and loaded, but it is visible momentarily beforehand.

File: js/pages/homePage.js [hunk 3]
```diff
@@ -167,107 +167,479 @@ export const whereAmIInDashboard = async () => {
...
+                     <li class="page-options--print">
+                        <a title="Print" aria-label="Print" href="#"></a>
+                     </li>
+                     <li class="page-options--email">
+                        <a title="Email" aria-label="Email" href="#"></a>
+                     </li>
+                  </ul>
+               </div>
+               <!-- END PAGE OPTIONS -->
+            </div>
```
Comment (JoeArmani) [inline js/pages/homePage.js:565]: remove this div - same as above

**Review by JoeArmani — CHANGES_REQUESTED** [review]
additional comments on the unused bullet points that are rendering at the tops of a couple pages

**Review by JoeArmani — APPROVED** [review]
looks good.

Only remaining comments:
UX: Icons on the 'What to expect' page look a bit skewed (too tall). Maybe these can be resized?
UX: It might make sense to have the images appear without text alongside them in the smaller (<700px) view.

Both of the above are probably nitpicks, but I wanted to mention them.

**Comment by JoeArmani** [comment]
> > looks good.
> > Only remaining comments: UX: Icons on the 'What to expect' page look a bit skewed (too tall). Maybe these can be resized? UX: It might make sense to have the images appear without text alongside them in the smaller (<700px) view.
> > Both of the above are probably nitpicks, but I wanted to mention them.
> 
> When I am trying to reduce the pixels the size of the image is reducing way to smaller so I left it as it is. I will re size the What to expect page images. Thank you.

Okay, just to reiterate: I don't think the images should be smaller in the mobile view. I just think the images should be on their own line (not alongside the text) in the mobile view. Thanks.

File: js/pages/homePage.js [hunk 4]
```diff
@@ -167,107 +167,411 @@ export const whereAmIInDashboard = async () => {
...
-    });
-  }
-
+   </div>
+</div>
+`;
+const signInBtn = df.querySelector('#signInBtn');
+const signUpBtn = df.querySelector('#signUpBtn');
+document.getElementById('signInWrapperDiv').replaceChildren(df);
+signInBtn.addEventListener('click', async () => {
```
Comment (anthonypetersen) [inline js/pages/homePage.js:542]: please fix the tabs on this section of code

**Review by anthonypetersen — CHANGES_REQUESTED** [review]

**Review by anthonypetersen — APPROVED** [review]
looks good

**Pull request merged**